data/strings/strings.txt

```diff
@@ -13687,7 +13691,7 @@
     ro = Deschide %1$@ la %2$@
     ru = Открывается в %1$@ в %2$@
     sk = Otvoria v %1$@ o %2$@
-    sr = Отвара се %1$@ у %2$@
+    sr = Отвара се у %1$@ у %2$@
```

Comment:
Is it ok that у is used twice?

Reply:
OMG
If you would write me what is - %1$@ 

and what is - %2$@
I would answer you. I translated a long time ago and asked you to insert it into the APK as soon as possible. And even if you made an APK for me. I look, check, correct and everything is as it should be.

Reply:
There is a mistake above in the en:
```
  [opens_dayoftheweek_at]
    tags = ios
    en = Opens %1$@ at %2$@
```
It should be `Opens on %1$@ at %2$@`, e.g. `Opens on Tuesday at 13:00`

Reply:
@ZeljkoBG the symbols like `%@?` and  `%1$@` are placeholders that get replaced with something. In this case we can assume it will be something like "Opens `Wednesday` at `12:00`"